app/routes/index/index.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  beforeModel() {
+  redirect() {
     this.transitionTo('index.mygoals');
   }
 });
```

Comment:
Since we have an `index` directory in routes, I though it was nicer to put it there rather than have an `index.js` file and an `index` directory, but that is pretty much the only reason. I'm not sure if there is a functional difference, but it appears to work the same way.